### PR TITLE
Add virtualenv for travis yml

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,3 +10,10 @@ install:
 
 before_script:
   - "flake8 ."
+
+branches:
+  only:
+    - master
+
+virtualenv:
+  system_site_packages: true
```
Comment (mknecht) [inline .travis.yml:19]: why do you need this?

Comment (admiral96) [inline .travis.yml:19]: So that is would use virtualenv 
noenv

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-flask=0.10.1
+flask
```
Comment (mknecht) [inline requirements.txt:1]: ah, why? fixed is usually a good idea, unless you have a reason not to.

**Comment by mknecht** [comment]
@admiral96 can you enumerate the things that your PR does? (hint: if it's more than one, split up)

**Comment by zkanda** [comment]
LGTM :+1:

**Pull request closed without merging**